app/src/main/res/layout/search_widget_extra_small_v2.xml

```diff
@@ -15,7 +15,7 @@
         android:layout_width="50dp"
         android:layout_height="50dp"
         android:layout_gravity="center"
-        android:contentDescription="@string/search_widget_content_description"
+        android:contentDescription="@string/search_widget_content_description_2"
```

Comment:
Shouldn't we remove the content description since we are setting this in `SearchWidgetProvider`? If we decide to remove this, we should go look for the other search_widget_content_description_2 usage.

Reply:
I would rather keep the .xml content descriptions instead of using tools:ignore.

Reply:
Okay. I don't think this was a blocker to me.  I saw you removed the content description in another place where we were setting content description code which is why I brought it up. Otherwise, it was more odd to use the string knowing there is a placeholder in it. Feel free to proceed with shipping.